valohai_yaml/schema/input-item.yaml

```diff
@@ -38,3 +38,10 @@ properties:
           - "suffix"
     default: false
     description: Whether to retain directories when using wildcards for this input.
+  edge-merge-mode:
+    type: string
+    enum:
+      - replace
+      - append
+    default: append
```

Comment:
The `default` here seems at odds with the Input object's `EdgeMergeMode.REPLACE`.